course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+{% verbatim %}
+
+<nav>
```

Comment:
Yup, that's what the `courseInstances` directive is there for...to hand off the cached course instance objects the SearchController already has in memory.